src/transactions/stake.js

```diff
@@ -1,10 +1,17 @@
 import algosdk from "algosdk";
 import { ids } from "./ids";
-import { setLoadingStage, microGARD, getMicroGardBalance } from "./lib"
+import { setLoadingStage, microGARD, getMicroGardBalance, getAppField, cdpInterest } from "./lib"
 import { accountInfo, getParams, signGroup, sendTxn } from "../wallets/wallets";
 
 const enc = new TextEncoder();
 
+export async function getStakingAPY(pool) {
```

Comment:
Maybe this function should be renamed since it's NL-specific.

Reply:
When we add more pools we'll update this fn (since it will be similar for all pools except NLL)